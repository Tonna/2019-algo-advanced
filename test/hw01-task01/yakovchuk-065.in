15
1 1 1 1 3 3 4 7 8 5 10 6 3 1
9
8 5
4 8
10 1
5 2
3 9
12 3
13 2
3 6
6 10
