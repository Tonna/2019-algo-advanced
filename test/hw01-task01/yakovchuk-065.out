8
15
8
6
17
13
9
10
21
